src/main/java/duke/response/DukeResponse.java

```diff
@@ -0,0 +1,36 @@
+package duke.response;
+
+import duke.DukeException;
+
+/**
+ * A response by Duke.
+ */
```

Comment:
I like that all the Coding practices were followed like class labels. Good job :)